File: Java Service/src/main/java/hello/Car.java
package hello;

public class Car {
	private final String name;
	private final String model;
	
	public Car(String name, String model) {
		this.name = name;
		this.model = model;
	}

	public String getName() {
		return name;
	}

	public String getModel() {
		return model;
	}

	public String toString() {
		return "Car [name=" + name + ", model=" + model + "]";
	}
	
}

File: DotNET Consumer/CarService/Model/Car.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CarService.Model
{
    public class Car
    {
        public string name { get; set; }
        public string model { get; set; }

        public string toString()
        {
            return $"Name: {this.name}; Model: {this.model}";
        }
    }
}
